### PR TITLE
Add Λnk README

### PR DESCRIPTION
**Pull request by Guardiola31337**
- Adds Λnk `README`

### CODE REVIEW

**Comment by pakoito** [comment]
Amazing job :D

File: modules/ank/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+./gradlew :module-name:runAnk
+```
+
+This will process all `*.md` files in `source`, write them to `target`, while providing all runtime dependencies (`sourceSets.main.runtimeClasspath`) in the `classpath`.
+
+## Errors output
+
+When something goes wrong Λnk shows what snippet has failed and where, including the compiler errors.
```
Comment (raulraja) [inline modules/ank/README.md]: This should probably show an example with a snippet that fails and how ank reports it.

File: modules/ank/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Modifiers
+
+By default Λnk compiles and evaluates the snippets of code included in `ank` sheds. However sometimes you might want a definition without printing the result out, or might want to replace an entire snippet with the output of the resulting evaluation. For these occasions Λnk provides a number of modifiers that you can add to the shed declaration.
```
Comment (raulraja) [inline modules/ank/README.md]: I would use ````kotlin:ank:*` fences instead of just `ank`

File: modules/ank/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+The language used (Kotlin or Java) should be prepended to `:ank` e.g. `kotlin:ank:replace`. The following modifiers are supported.
+
+| Modifier | Explanation |
+|---|---|
+| `:silent` | Suppresses output; under this modifier the input and output text are identical. |
```
Comment (raulraja) [inline modules/ank/README.md]: We should provide an example for each one of these.

**Pull request merged**